ui/src/lib/forms/feedingInformation.js

```diff
@@ -0,0 +1,82 @@
+import * as yup from 'yup';
+
+const feedingInfoFields = {
+  'Feeding Information': [
+    {
+      name: 'breastFeeding',
+      label: 'Breastfeeding:',
+      type: 'radio',
+      validate: yup.string().required('Breastfeeding status is required'),
+      width: {
+        xs: 12,
+        sm: 14,
```

Comment:
Change the width for "sm" devices to 12, "md" to 8, and "lg" to 6 for all the fields